runtime/pendulum/src/lib.rs

```diff
@@ -468,8 +475,9 @@ impl pallet_balances::Config for Runtime {
 	type ReserveIdentifier = ReserveIdentifier;
 	type FreezeIdentifier = ();
 	type MaxFreezes = ();
-	type MaxHolds = ConstU32<1>;
+	type MaxHolds = ConstU32<2>;
```

Comment:
Otherwise, we get an error `MaxHolds should be greater than or equal to the number of hold reasons: 1 < 2`, which is an internal check of the pallet.
Apparently the `RuntimeHoldReason` has 2 variants.